room.go

```diff
@@ -194,6 +194,19 @@ func (r *Room) JoinWithToken(url, token string, opts ...ConnectOption) error {
 func (r *Room) Disconnect() {
 	_ = r.engine.client.SendLeave()
 	r.engine.Close()
+
+	var localPubs []*LocalTrackPublication
+	r.LocalParticipant.tracks.Range(func(_, value interface{}) bool {
+		track := value.(*LocalTrackPublication)
+		if track.Track() != nil {
+			localPubs = append(localPubs, track)
+		}
+		return true
+	})
+
+	for _, pub := range localPubs {
+		pub.CloseTrack()
+	}
```

Comment:
Can this be moved to a method in `LocalParticipant`? Similar to `republishTracks`. Would be good not to `Range` on an internal element of a different structure from here.

Reply:
Done